moped-database/migrations/1651001224000_alter_project_list_view_funding/up.sql

```diff
@@ -0,0 +1,96 @@
+drop view project_list_view;
+
+CREATE VIEW public.project_list_view AS
```

Comment:
## TLDR
This query is very fast.

## Considerations
Related to our quick conversation about SQL this morning, I am going to make a point to start to be more aware of execution times for queries that are being embedded as views (just like we would for a graphql query.)

One of the very first things we need to consider in the pursuit of a snappy, responsive app is a snappy, responsive database as it is our data source for like almost everything. To quantify a query's performance, I want to propose that we can use execution time on the read replica as a way to make apples to apples comparisons and track query performance over time.

I say this because:

1) both the RR and the production database operate on the exact same data at any given time -- this is the nature of the read replica, and
2) the RR is a lessor powered instance that the production database. If it's fast to compile and execute on the RR, it should be at least that fast on the real deal.

Caveat: This tendency will break down if, say, the production database were under continual load above what it is capable of keeping up with, but we are _very, very_ far from that being the case right now. Historical load graphs are available in the RDS section of AWS which we can use to ensure that remains the case.

Anyway, all that said, when I ask postgres to `EXPLAIN ANALYZE` this query for me, it shows that the database is able to find and prepare the results to this query in

<img width="209" alt="Screen Shot 2022-05-05 at 3 10 11 PM" src="https://user-images.githubusercontent.com/3653206/167017681-5c5676e3-a57f-4957-a5c1-7b4bd5a86200.png">

That's super-fast in my opinion.⚡🔥 That's down on the order of local network latency let alone the internet, so that's really flying.